.github/workflows/sycl-detect-changes.yml

```
@@ -57,6 +57,19 @@ jobs:
               - devops/scripts/install_drivers.sh
             perf-tests:
               - sycl/test-e2e/PerformanceTests/**
+            esimd-tests:
```

```suggestion
            esimd:
```

to align with convention in this file.